sources/packages/backend/libs/integrations/src/esdc-integration/student-loan-balances/student-loan-balances.integration.service.ts

```diff
@@ -34,7 +40,10 @@ export class StudentLoanBalancesIntegrationService extends SFTPIntegrationBase<S
         `The Student loan balances file ${remoteFilePath} has an invalid record type on header: ${header.recordTypeCode}.`,
       );
       // If the header is not the expected one, throw an error.
-      throw new Error("Invalid file header.");
+      throw new CustomNamedError(
+        "Invalid file header.",
```

Comment:
Error description can be more precise. `Invalid record type <record_type> on header.`